modules/ROOT/pages/policy-mule4-jwt-validation.adoc

```diff
@@ -82,15 +88,15 @@ For Mule applications, the following parameters are displayed based on whether y
 |===
 | Element | Description | Example
 | JWT Origin | Specifies from where in the request the JWT will be extracted:
-* Bearer Authentication Header
+* HTTP Bearer Authentication Header
 * Custom Expression
 
-If you set it to Bearer Authentication Header, the JWT will be expected as Bearer. +
-If you set this field to *Custom Expression*, a DataWeave Expression returning the token must be provided.
+If you set it to HTTP Bearer Authentication Header, the JWT is expected as Bearer. +
```

Comment:
To what does "it" refer?
And if you are "setting" it using syntax, then it should be formatted as user-supplied syntax.

Reply:
Done! Thanks!